src/View/Helper/FormHelper.php

```diff
@@ -601,12 +622,19 @@ public function secure(array $fields = [], array $secureAttributes = [])
             'value' => $tokenData['unlocked'],
         ]);
         $out .= $this->hidden('_Token.unlocked', $tokenUnlocked);
+        if (!empty($tokenData['optional'])) {
+            $tokenOptional = array_merge($secureAttributes, [
```

Comment:
Indeed, those are all kind of the inferior operator. But this is then out of scope of this PR and can be cleaned up later.